### ISSUE TITLE
Please provide the source code

### ISSUE THREAD

**Issue by k0np4ku** [1354961639/AzurLane-Scripts-Autopatcher-beta#3]
Hello, sorry for coming in and intruding you out of nowhere.

I see you are distributing the modified autopatcher without providing the source code.

Frankly, I don't really care about azur lane. My code is shit, and I just restored my github account back to life like 3 days ago.

I have never decided its license, but all rights reserved to the original author, regardless. And I'm thinking to use GNU GPLv3 for sharing improvements. Which means, whoever uses the original source are not permitted to distribute a closed-source version.

TL;DR: Provide the source code or delete this repository.

Hopefully we can resolve this peacefully without me needing to request a takedown to Github.

Thank you.

Best regards.
- k0np4ku
